code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -467,7 +467,9 @@
 		if(H.physiology)
 			H.physiology.stamina_mod *= 0.5
 		if(H.dna && H.dna.species)
-			H.dna.species.punchdamagehigh *= 5
+			H.dna.species.punchdamagehigh *= 4
+			H.dna.species.punchdamagelow  *= 10
```

Comment:
what if this makes the low higher than the high

Reply:
GOOD POINT LEMME DO THE SANE THING AND LOWER IT SO WE CAN KEEP THAT FROM HAPPENING. SINCE I THINK MAYBE MILITARY SYNTHS WOULD END UP WITH WAY MORE LOW THAN HIGH AND THAT'S GONNA CAUSE ISSUES.

**ALSO, PUSH THAT CAPSLOCK BUTOTN.**